hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/file/negative/AppendFailuresSpec.java

```diff
@@ -55,21 +54,12 @@ private HapiSpec handleRejectsOversized() {
         return defaultHapiSpec("handleRejectsMissingWacl")
                 .given(
                         getFileContents(APP_PROPERTIES).saveTo("tmp-application.properties"),
-                        fileUpdate(APP_PROPERTIES)
-                                .payingWith(ADDRESS_BOOK_CONTROL)
-                                .overridingProps(Map.of("maxFileSize", "3")))
+                        overriding("files.maxSizeKb", "3"))
                 .when(
                         fileCreate("file").contents(BYTES_3K_MINUS1),
                         fileAppend("file").content(BYTES_1),
                         fileAppend("file").content(BYTES_1).hasKnownStatus(MAX_FILE_SIZE_EXCEEDED))
-                .then(
-                        fileUpdate(APP_PROPERTIES)
-                                .payingWith(ADDRESS_BOOK_CONTROL)
-                                .overridingProps(Map.of("maxFileSize", "1024"))
-                                .droppingUnmentioned(),
-                        fileUpdate(APP_PROPERTIES)
-                                .payingWith(ADDRESS_BOOK_CONTROL)
-                                .path("tmp-application.properties"));
+                .then(overriding("files.maxSizeKb", "1024"));
```

Comment:
Why do these lines need to be removed? What caused them to fail? I'm worried about whether modifying the tests either diminishes the test coverage or if it breaks mono testing.

Reply:
Those `fileUpdate(APP_PROPERTIES)` operations were very early versions of the `overriding()` utility; and until we switched genesis schema to have empty `0.0.121`, they prevented this test from running at all.

Semantics of this test haven't changed.

Reply:
We need to use the overriding operation in case 121 file containing all properties otherwise, you cannot use the single file update.

Reply:
`overriding()` is the new version of `fileUpdate(APP_PROPERTIES).overridingProps()`---it is resilient to file `0.0.121` having a size >6kb.